Casks/w/wins.rb

```diff
@@ -0,0 +1,29 @@
+cask "wins" do
+  version "1.6.6"
+  sha256 "594fb383c26f3b36dfa132312ddbafb0e59f7566616b5f02294e769cecf55605"
+
+  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-#{version}.dmg", verified: "backblazeb2.com/file/winsWebsite/"
```

Comment:
`verified` on its own line, indented, and should contain base components of the `url`
```suggestion
  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-#{version}.dmg",
      verified: "f005.backblazeb2.com/file/winsWebsite/"
```